### DISCUSSION TITLE
SSL Connection Error

### DISCUSSION THREAD

**Discussion by chetak800** [visose/Robots#223] [Q&A] [answered]
Hello There ! , I am Currently using Rhino 7 with Robots Library , but when i try to load robot the library show's me the following error![image](https://user-images.githubusercontent.com/63487856/226815446-6ab93b9a-19e7-4fd8-a6b0-88c9918cd99e.png)
i am Using Macbook Air M1 with Mac os Ventura 

Can you please help me with error ?

**Comment by visose** [comment, marked as answer]
Closing this old install/library-download thread with the most likely answer. SSL errors on Rhino 7/macOS library downloads are usually an environment/package-manager/network certificate issue rather than a robot library issue. Update Rhino/Robots and retry the Package Manager/Libraries UI; if online library access is blocked, manually place the matching `.xml`/`.3dm` pair from the `libraries` branch into the user's `Documents/Robots` folder. Please reopen with current Rhino, macOS, Robots version, and the exact error if it still reproduces.
